src/scripts/main.js

```diff
@@ -1,3 +1,20 @@
 'use strict';
 
-// write your code here
+const elements = [...document.getElementsByClassName('population')];
+const totalElement = document.querySelector('.total-population');
+const averageElement = document.querySelector('.average-population');
+const newArray = [];
+let totalPopulation = 0;
+let average = 0;
+
+elements.forEach((element) => {
+  const number = element.textContent.trim();
```

Comment:
Consider using `parseInt` or `parseFloat` instead of `Number` for converting the string to a number, as it makes the intention clearer when dealing with numeric strings.